stable/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 1.10.1
+version: 1.10.2
```

Comment:
This added functionality is worth a minor version bump ;-)

Reply:
Oh yeah, so true! done!